clients/src/main/java/org/apache/kafka/common/requests/MetadataResponse.java

```diff
@@ -44,6 +44,9 @@
     private static final String CONTROLLER_ID_KEY_NAME = "controller_id";
     public static final int NO_CONTROLLER_ID = -1;
 
+    private static final String CLUSTER_ID_KEY_NAME = "cluster_id";
+    public static final String NO_CLUSTER_ID = null;
```

Comment:
We should probably not bother with this. We use constants like this when using magic values like `-1`. For nullable values, it's fine to just use plain `null`. Particularly as this constant lives in an internal class, so users of `ClusterResource` can't use it anyway.